client/asset/btc/btc.go

```diff
@@ -97,24 +99,39 @@ var (
 			DefaultValue: "8332",
 		},
 		{
-			Key:          "fallbackfee",
-			DisplayName:  "Fallback fee rate",
-			Description:  "Bitcoin's 'fallbackfee' rate. Units: BTC/kB",
+			Key:         "fallbackfee",
+			DisplayName: "Fallback fee rate",
+			Description: "The fee rate to use for fee payment and withdrawals when" +
+				" estimatesmartfee is not available. Units: BTC/kB",
 			DefaultValue: defaultFee * 1000 / 1e8,
 		},
 		{
-			Key:          "redeemconftarget",
-			DisplayName:  "Redeem confirmation target",
-			Description:  "The target number of blocks for the redeem transaction to get a confirmation. Used to set the transaction's fee rate. (default: 2 blocks)",
+			Key:         "feeratelimit",
+			DisplayName: "Highest acceptable fee rate",
+			Description: "This is the highest network fee rate you are willing to " +
+				"pay on swap transactions. If you set this too low, you may not be " +
+				"able to place orders with servers that allow for higher swap rates as" +
+				" network conditions demand. Units: sats/byte",
```

Comment:
[My previous comment](https://github.com/decred/dcrdex/pull/874#discussion_r545429003) was resolved and maybe a little unclear on my position, but there is now a discrepancy between the `fallbackfee` units and the `feeratelimit` units. For consistency, I do think we should use the same units for both.

Reply:
I'm aware of the consistency consideration, but I feel strongly that feeratelimit should be integer units like sat/bytes.  The consideration is how these settings are used.  fallbackfee matches the node and RPC settings there it takes e.g. btc/kb, while the limit corresponds to e.g. sat/byte as per server setting.  I would must rather have these settings changable without mental gymnastics than have them be consistent.

Reply:
Hmm, I guess I have to correct myself.  `fallbackFeeRate` is converted to integer too.  Ugh, well I *hate* the decimal units and all the prefixed zeros, but changing fallbackfee units is too disruptive, so that's not changing now.  I especially dislike btc/kb for this limit since it is not applied in that format anywhere, but if we insist on these two settings have consistent units, Ok.  However, I do not thing consistency between these two settings is a more powerful consideration that clarity and easy of use.

Reply:
Updated the message, @buck54321 are you convinced regarding the integer units ?

Reply:
Something to consider: UI could eventually have a units dropdown similar to the Bitcoin Core Qt interface:

![image](https://user-images.githubusercontent.com/9373513/103158022-ef3aed80-477e-11eb-8455-95d852028e3b.png)

So it was made that way so user could specify units anyway they like, the question then becomes "what is the natural/native unit to use on the backend?"

Reply:
> I especially dislike btc/kb for feeratelimit since it is not applied in that format anywhere and the server config (maxfeerate) is in unit/byte

Those seem like implementation details. As something that hits the UI, the unit chosen should be the one that most users expect or are comfortable with. I also personally think that atoms/byte is the best unit to use for fees, but atoms/kB was the UI unit chosen by bitcoind and dcrwallet, and even in that screenshot from Bitcoin Qt, the **per kB** part is not selectable. I think we're losing this battle.

> However, I do not think consistency between these two settings is a more powerful consideration than clarity and easy of use.

Consistency is not mutually exclusive with clarity and ease of use, and it's easy to argue that different units for the same thing on the same form is a formula for bad UX. What's clear and easy is typically what's expected or conventional, no?


More than anything, I just don't care to fret about this any more. If you're adamant about atoms/byte, go for it.

Reply:
Well I was actually trying to talk myself out of this hangup with the possiblity for GUI providing a units selector say some point.  If that seems reasonable to implement at some point I no longer have an issue with starting this with with consistent units on the GUI, if somewhat difficult to reconcile with the backend and corresponding units.
See any reason that couldn't be achieved at some point in GUI @buck54321 ?

Reply:
I can see that. Maybe as part of a larger solution to solve our hard-coded `1e8` problem, which will need to be fixed for eth. The backends can define conventional and alternative units, each with their own divisor.

Reply:
Like the sound of that comprehensive solution 😻